biz.aQute.bndlib.tests/src/test/MacroTest.java

```diff
@@ -1121,21 +1121,23 @@ public static void testPackagesMacro() throws Exception {
 				new Jar(IO.getFile("bin"))
 		});
 		b.setProperty("Private-Package",
-				"test.packageinfo.annotated,test.packageinfo.notannotated,test.packageinfo.nopackageinfo");
-
+				"test.packageinfo.annotated,test.packageinfo.notannotated,test.packageinfo.nopackageinfo,test.activator");
 		b.setProperty("All-Packages", "${packages}");
 		b.setProperty("Annotated", "${packages;annotated;test.packageinfo.annotated.BlahAnnotation}");
 		b.setProperty("Named", "${packages;named;*.notannotated}");
 		b.setProperty("Negated", "${packages;named;!*.no*}");
+		b.setProperty("Exported", "${packages;exported}");
 		b.build();
 
 		assertEquals(0, b.getErrors().size());
 
-		assertEquals("test.packageinfo.annotated,test.packageinfo.notannotated,test.packageinfo.nopackageinfo",
+		assertEquals(
+				"test.packageinfo.annotated,test.packageinfo.notannotated,test.packageinfo.nopackageinfo,test.activator",
 				b.getProperty("All-Packages"));
 		assertEquals("test.packageinfo.annotated", b.getProperty("Annotated"));
 		assertEquals("test.packageinfo.notannotated", b.getProperty("Named"));
-		assertEquals("test.packageinfo.annotated", b.getProperty("Negated"));
+		assertEquals("test.packageinfo.annotated,test.activator", b.getProperty("Negated"));
+		assertFalse(b.getProperty("Exported").contains("test.activator"));
```

Comment:
This should be an assertEquals which will provide a useful message when there is a failure. assertFalse doesn't provide any useful message.